test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -137,6 +146,24 @@ func describeQatDpdkPlugin() {
 			gomega.Expect(err).To(gomega.BeNil(), utils.GetPodLogs(ctx, f, "qat-dpdk-test-crypto-perf", "crypto-perf"))
 		})
 
+		ginkgo.It("deploys a crypto pod (qat-engine testapp) [App:qat-engine]", func(ctx context.Context) {
+			command := []string{
+				"testapp",
+				"-engine", "qathwtest",
+				"-async_jobs", "1",
+				"-c", "1",
+				"-n", "1",
+				"-nc", "1",
+				"-v",
+				"-hw_algo", "0x0029",
+			}
+			pod := createPod(ctx, f, "qat-engine-testapp", resourceName, "intel/openssl-qat-engine:devel", command)
+
+			ginkgo.By("waiting the qat-engine-testapp pod for the resource" + resourceName.String() + "to finish successfully")
```

Comment:
do you want all the test pod to be named the same or is that just a copy-paste error?

Reply:
copy-paste error.
I changed before, but I think while making `createPod` I copied wrongly.